components/script/dom/response.rs

```diff
@@ -100,34 +99,29 @@ impl Response {
             ));
         }
 
-        // Step 3
         let r = Response::new(global);
 
-        // Step 4
+        // Step 3
         *r.status.borrow_mut() = Some(StatusCode::from_u16(init.status).unwrap());
 
-        // Step 5
+        // Step 4
         *r.raw_status.borrow_mut() = Some((init.status, init.statusText.clone().into()));
 
-        // Step 6
+        // Step 5
         if let Some(ref headers_member) = init.headers {
-            // Step 6.1
-            r.Headers().empty_header_list();
```

Comment:
this is mentioned as a no-op and removed in https://github.com/whatwg/fetch/commit/90cd540dc2a25df48b85a361f49901ac0835957b so I just removed it along with the step comments update commit.